server/src/main/java/com/cloud/api/query/vo/VpcOfferingJoinVO.java

```diff
@@ -197,6 +200,10 @@ public String getZoneName() {
         return zoneName;
     }
 
+    public String getInternetProtocol() {
+        return internetProtocol;
```

Comment:
@shwstppr 
advice: return internet protocol= "IPv4"  in vpc offering response by default ?